benchmarks/benchmark_init.cc

```diff
@@ -0,0 +1,94 @@
+#include "benchmark_common.h"
+#include "config_utility.h"
+#include "constants.h"
+#include "err_constants.h"
+#include "live_model.h"
+#include "model_mgmt.h"
+
+#include <benchmark/benchmark.h>
+
+#include <algorithm>
+#include <iostream>
+
+namespace r = reinforcement_learning;
+namespace u = reinforcement_learning::utility;
+namespace err = reinforcement_learning::error_code;
+namespace cfg = reinforcement_learning::utility::config;
+
+const auto JSON_CFG = R"(
+{
+  "ApplicationID": "rnc-123456-a",
+  "IsExplorationEnabled": true,
+  "InitialExplorationEpsilon": 1.0
+}
+)";
+
+template <class... ExtraArgs>
+static void bench_init(benchmark::State& state, ExtraArgs&&... extra_args)
+{
+  int res[sizeof...(extra_args)] = {extra_args...};
+  auto load_model = res[0];
+  auto model_id = res[1];
+
+  std::ostringstream model_path;
+  model_path << "benchmarks/models/";
+  switch (model_id)
+  {
+    case 0:
+      model_path << "cb_small.m";
```

Comment:
Wondering if adding the model command line in a comment here would be useful as reference.

Reply:
added comment, only uses --cb_explore_adf (and -b 18 by default)